CelebrityStalker.xcodeproj/project.pbxproj

```diff
@@ -157,6 +160,7 @@
 			files = (
 				3FFB72E91BD9273B000E3300 /* Main.storyboard in Resources */,
 				3F0ACCD11BD5664400E3BDE1 /* LaunchScreen.storyboard in Resources */,
+				A1C9FAA21BD9369300161C68 /* celebBio.JSON in Resources */,
```

Comment:
*And again, the __JSON file__, here being added to the __Build Phase__ so it will be compiled at run time.*